### ISSUE TITLE
p-chip: aria-label is not applied to remove icon

### ISSUE THREAD

**Issue by akashuwtw** [primefaces/primeng#14973]
### Describe the bug

`aria-label` set to `p-chip` are not getting applied to it's inner `<span>` element of remove icon.

The result of this is that the screen reader doesn't reads anything when remove icon receives a keyboard focus.

### Environment

OS: Windows 10

### Angular version

16.2.7

### PrimeNG version

16.5.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Browser(s)

Chrome 121.0.6167.161, Edge 121.0.2277.112

### Steps to reproduce the behavior

```
<p-chip
    *ngFor="let file of files"
    label="{{ file.name }}"
    [attr.aria-label]="getRemoveFileLabel(file.name)"
    class="sm"
    [removable]="true"
    removeIcon="pi cross"
    (onRemove)="onRemoveFile(file)" />
```

### Expected behavior

`aria-label` should be applied to the `<span>` of remove icon which actually gets the keyboard focus.

**Comment by cetincakiroglu** [comment]
Fixed in https://github.com/primefaces/primeng/pull/15034
